src/pages/VeSDL/VeTokenCalculator.tsx

```diff
@@ -39,51 +70,60 @@ export default function VeTokenCalculator({
         <Stack direction="column" spacing={3}>
           <TextField
             label={t("totalVeSdl")}
-            value={totalVeSdl}
-            onChange={(e) =>
-              readableDecimalNumberRegex.test(e.target.value) &&
-              setTotalVeSdl(e.target.value)
-            }
+            value={formatBNToString(totalSupplyVeSdl, 18)}
             fullWidth
           />
           <Divider />
           <Typography variant="subtitle1">{t("maxBoostCalculator")}</Typography>
 
           <TextField
             variant="standard"
-            value={poolName}
+            value={poolNameValue}
             placeholder={t("selectPoolName")}
-            onChange={(e) => setPoolName(e.target.value)}
+            onChange={(e) => setPoolNameValue(e.target.value)}
             select
             SelectProps={{ IconComponent: ArrowDownIcon }}
           >
             <MenuItem value="">
               <em>None</em>
             </MenuItem>
-            <MenuItem value="D4">D4 - Ethereum</MenuItem>
-            <MenuItem value="BTC">BTC - Ethereum</MenuItem>
-            <MenuItem value="tBTC">tBTC - Ethereum</MenuItem>
-            <MenuItem value="arbFrax">arbFrax - Arbitrum</MenuItem>
+            {Object.keys(POOLS_MAP).map((poolName) => (
```

Comment:
pools hook instead of POOLS_MAP as we're currently removing all instance of POOLS_MAP